### DISCUSSION TITLE
Usability for Boards

### DISCUSSION THREAD

**Discussion by dclear** [community/community#4446] [Projects and Issues]
Very excited about the new issues beta and can't wait to play with it. What I ask for here may be accounted for. 2 requests, related to making boards usable.

1. Add navigation to create an issue from the board view
<img width="608" alt="Screen Shot 2021-07-01 at 10 51 51 AM" src="https://user-images.githubusercontent.com/1197912/124161657-e8a56f80-da5a-11eb-81ac-f01ec61d2f8e.png">

2. Improve draggable column ordering. It is pretty frustrating to work with. Maybe make them collapsible to help with repositioning.

Ty!

**Comment by sfullerbeckman** [comment]
I definitely agree with your second request. Making them collapsible would be very helpful or at least finding some way of simplifying the drag behavior.

**Comment by dclear** [comment]
1. above is somewhat related to https://github.com/github/feedback/discussions/4276. Issue types on boards and visually revealing their relationships to other issues and PRs, is critical, in my view.

I personally don't get much value from `add a card` ; that is really `add an existing issue`.
